src/models/childrenModel.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+
+const childrenSchema = new mongoose.Schema({
+    userName: {
+        type: String,
+        required: true
+    },
+    password: {
+        type: String,
+        required: [true, 'Please provide a password'],
+        minLength: [6, 'Password must be at least 6 characters long'],
+        trim: true,
+    },
+    tasks: [
+        {
+            taskId: {
+                type: mongoose.Schema.ObjectId,
+                ref: 'Tasks',
+            },
+            isCompleted: {
+                type: false,
+            }
+        }
+    ],
+    rewards: [
+        {
+            rewardId: {
+                type: mongoose.Schema.ObjectId,
+                ref: 'Rewards',
+            },
+            receivedAt: {
+                type: Date
+            }
+        }
+        
+    ],
+    image: {
+        type: String,
+        default: 'default.jpg',
+    },
+});
```

Comment:
we may add `totalRewardPoints` or similar to hold the accumulated points for each child.

Reply:
added totalPoints field to children model